crux_core/src/bridge/mod.rs

```diff
@@ -7,7 +7,8 @@ use serde::{Deserialize, Serialize};
 use thiserror::Error;
 
 use crate::{core::ResolveError, App, Core};
-use registry::{EffectId, ResolveRegistry};
+pub use registry::EffectId;
```

Comment:
Type was already semi-public as part of `EffectFfi` I think, this just makes it explicit.